### ISSUE TITLE
YOLOv2 Loss function modification.   

### ISSUE THREAD

**Issue by HemaZ** [thtrieu/darkflow#916]
hello,
I was wonder how to modify the loss function of YOLO v2. 
I want to remove the term responsible for object classification. 
![image](https://user-images.githubusercontent.com/6087928/46485567-346c5100-c7fc-11e8-94be-140a7cb22f7b.png)
the last term. 

i looked at this file darkflow/net/yolov2/train.py but I couldn't figure it out.

**Comment by sandeeprepakula** [comment]
Can you try a quick hack by making all labels as single constant label.
